pkg/config/remote/configs_apm_sampling.go

```diff
@@ -0,0 +1,57 @@
+package remote
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+)
+
+// APMSamplingConfig is an apm sampling config
+type APMSamplingConfig struct {
+	Config
+	pb.APMSampling
+}
+
+// APMSamplingUpdate is an apm sampling config update
+type APMSamplingUpdate struct {
+	Config *APMSamplingConfig
+}
+
+type apmSamplingConfigs struct {
+	config *APMSamplingConfig
+}
+
+func newApmSamplingConfigs() *apmSamplingConfigs {
+	return &apmSamplingConfigs{}
+}
+
+func (c *apmSamplingConfigs) update(configFiles map[string]configFiles) (*APMSamplingUpdate, error) {
```

Comment:
probably we should pass around the configuration IDs as UUIDs to ensure they match the format and have a packed representation

Reply:
I don't have a supper string opinion but not enforcing them agent side sounds ok to me.